odkx-src/sync-endpoint.rst

```diff
@@ -24,6 +24,8 @@ ODK-X Sync Endpoint
 It communicates with your ODK-X Android applications to synchronize
 your data and application files.
 
+There is an important library called :dfn:`Sync Client` which implements the sync protocol by interacting with Sync Endpoint and other tools. The source of this library can be found here: `https://github.com/odk-x/sync-client <https://github.com/odk-x/sync-client/>`_. This is a library that can be used to execute the ODK-X sync protocol natively on non-Android systems.
+
```

Comment:
Instead of saying "non-Android systems," it is better to say in Java applications. Also technically 'natively', is not correct you need a JVM (not native to the processor)

Reply:
Yes, we need a Java Virtual Machine; updated it!